specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -4221,13 +4229,28 @@
       "description": "Session host configurations of HostPool.",
       "type": "object",
       "properties": {
+        "provisioningState": {
```

Comment:
I know this property is suggested for async operation; but it is weird for our case, as we are not doing any provisioning for session host configuration subresource, we are doing validation.  I think we don't really need to validate session host configuration during update api call.  For example, we update session host configuration when add vm to existing host pool scenario; it is similar to any DSC extension.  It will fail during deployment if the script failed or doesn't exists.

Reply:
for update and create call we validate the Session Host Configuration. We talked about this in the meeting yesterday.